webots_ros2_driver/src/WebotsNode.cpp

```diff
@@ -175,9 +176,9 @@ namespace webots_ros2_driver
       if (!pluginElement->Attribute("type"))
         throw std::runtime_error("Invalid URDF, a plugin is missing a `type` property at line " + std::to_string(pluginElement->GetLineNum()));
 
-      const std::string type = pluginElement->Attribute("type");
+      std::string type = pluginElement->Attribute("type");
```

Comment:
Can't we keep it const?
```suggestion
      const std::string type = pluginElement->Attribute("type");
```

Reply:
Yes, added `const`